skbio/stats/power.py

```diff
@@ -908,11 +924,131 @@ def _compare_distributions(test, samples, counts=5, mode="ind", num_iter=1000):
             subs = [np.random.choice(np.array(pop), counts[i], replace=False)
                     for i, pop in enumerate(samples)]
 
-        p_values[idx] = test(subs)
+        p_values[:, idx] = test(subs)
+
+    if num_p == 1:
+        p_values = np.hstack(p_values)
 
     return p_values
```

Comment:
Noticed that the doc-string hasn't been updated to reflect the change to `p_values`.
It would be nice to mention this change (i.e. the returned `p_values` array has dimensions `num_p` and `num_iter`)